src/search_algo.hpp

```diff
@@ -239,24 +240,33 @@ void loadDbIndexFromDisk(
 
     double finish = sysTime() - start;
     myPrint(options, 1, " done.\n");
+
+
+    myPrint(options, 2, "    # original subjects:   ", globalHolder.indexFile.seqs.size(), "\n");
+    myPrint(options, 2, "    # translated subjects: ", globalHolder.transSbjSeqs.size(), "\n");
+    myPrint(options, 2, "    # reduced subjects:    ", globalHolder.redSbjSeqs.size(), "\n");
+    bool const indexHasSTaxIDs = globalHolder.indexFile.sTaxIds.size() == globalHolder.indexFile.seqs.size();
+    myPrint(options, 2, "    has taxonomic IDs:     ", indexHasSTaxIDs, "\n");
+    bool const indexHasTaxTree = globalHolder.indexFile.taxonNames.size() >= globalHolder.indexFile.seqs.size();
+    myPrint(options, 2, "    has taxonomic tree:    ", indexHasTaxTree, "\n");
     myPrint(options, 2, "Runtime: ", finish, "s \n\n");
 
-    // this is actually part of prepareScoring(), but the values are just available now
-    // TODO actually this should be for all modes where total size of redSbjSeqs != origSeqs
```

Comment:
Changing this has (expectedly) changed the e-values for tblastn mode, because the total size of reduced subject sequences is now always used for length-normalisation of the e-values (whereas before tblastn incorrectly used original sequence lengths)